mikupad.html

```diff
@@ -2124,6 +2656,10 @@
 					penalize_nl: penalizeNl,
 					ignore_eos: ignoreEos,
 				} : {}),
+				// not sure if I need the openaiPresets here
+				...(!openaiPresets || [0,2].includes(endpointAPI) && logitBias.bias.length > 0 ? {
+					logit_bias: logitBiasParam, // [[39,-20],[37934,-20]], //
+				} : {}),

```

Comment:
You shouldn't need it since logit_bias is supported by the official OpenAI endpoint.
But a conversion will be necessary since OpenAI uses a map-like format (e.g. `{6342:-10, 1582:-10}`).
```suggestion
				// not sure if I need the openaiPresets here
				...(logitBias.bias.length > 0 ? {
					logit_bias: logitBiasParam, // [[39,-20],[37934,-20]], //
				} : {}),
```